client/plots/matrix/hierCluster.renderers.js

```diff
@@ -10,6 +10,11 @@ export function maySetSandboxHeader() {
 		case TermTypes.METABOLITE_INTENSITY:
 			this.dom.header.text('Metabolite Intensity Clustering')
 			break
+		case termTypes.NUMERIC_DICTIONARY_TERM:
```

Comment:
this should be uppercase `TermTypes`, I get an error when I try to load `Drug Sensitivity` chart.

Reply:
thanks just fixed it